src/main/java/jalau/at18/katas/bankocr/daniela/Zero.txt

```diff
@@ -0,0 +1,3 @@
+ _   _
```

Comment:
This could go to resources folder

Reply:
this file It was an initial way to see how I was going to start the code but I'm not using it, I will erase it. :)